src/modules/pos/validator/components/ValidatorProfile/DetailsView.js

```diff
@@ -76,13 +88,43 @@ const DetailsView = ({ data, isMyProfile }) => {
             <div className={`${grid.col} ${styles.item}`}>
               <div className={`${styles.title} ${theme}`}>
                 <span>{label} </span>
-                {onEdit && typeof onEdit === 'function' && (
-                  <button onClick={onEdit} className={styles.editBtn}>
-                    <Icon name="editActiveIcon" />
-                  </button>
+                {onEdit &&
+                  typeof onEdit === 'function' &&
+                  (hasChangeCommission ? (
+                    <Tooltip
+                      className={classNames(styles.editDisabledIcon, styles.editBtn)}
+                      tooltipClassName={`${styles.tooltipClassNameProp}`}
+                      size="maxContent"
+                      position="bottom"
+                      content={<Icon name="editDisabled" />}
+                    >
+                      <p>
+                        You have to wait for your current commission change to finalize before you
+                        can edit again.
+                      </p>
+                    </Tooltip>
+                  ) : (
+                    <button
+                      onClick={onEdit}
+                      className={styles.editBtn}
+                      disabled={hasChangeCommission}
+                    >
+                      <Icon name="editActiveIcon" />
+                    </button>
+                  ))}
+              </div>
+              <div
+                className={classNames(
+                  styles.value,
+                  styles.capitalized,
+                  hasChangeCommission &&
+                    onEdit &&
+                    typeof onEdit === 'function' &&
+                    styles.textLineThrough
```

Comment:
I think it would be more readable if you use classNames like this. What do you think

```suggestion
                    styles.value,
                    styles.capitalized,
                    { 
                      [styles.textLineThrough] : hasChangeCommission && onEdit && typeof onEdit === 'function'
                    }
```

Reply:
👍 I agree when there are conditional classnames that is how we should do, https://github.com/LiskHQ/lisk-desktop/pull/5092/commits/200b2551d1c4c8f074ca87fc49ba01a3f5ab0e61